examples/localai-webui/README.md

```diff
@@ -0,0 +1,26 @@
+# localai-webui
```

Comment:
shall this be localai-chat instead?

Reply:
I think WebUI is more appropriate, chat is covering all the other projects too?